CHANGELOG.md

```diff
@@ -1,3 +1,265 @@
+# 23.03 (2023-03-31)
```

Comment:
@the-mikedavis I noticed every time you had to write this by looking at the logs, even though writing it from scratch will be better for the reader, but it is a lot of work, should we add an unreleased section so each new feature change needs to be written to that so you don't have to spend so much time on this?

Reply:
see https://github.com/helix-editor/helix/pull/4150 for previous discussion. I think the points raised there were pretty good. I could comeup with a solution which would somewhat automate it but I guess Mikes point about that being an unnecessary effort still applies